linode/manager
# refactor: [M3-7449] - Dismissible Banner v7 story migration

## Description 📝
- MIgrated DismissibleBanner component story to a V7 (.tsx) story

## Changes  🔄
- migrated story
- added tests
- removed index file + updated exports

## How to test 🧪

### Prerequisites
- pull code
- yarn && yarn storybook

### Verification steps 
- checkout http://localhost:6006/?path=/docs/components-notifications-dismissible-banners--documentation + confirm story loads properly
- Confirm no regression for DismissibleBanner on CM
- `yarn test DismissibleBanner/DismissibleBanner.test.tsx`

## As an Author I have considered 🤔

*Check all that apply*
